### PR TITLE
Fix invalid input

### PR DESCRIPTION
**Pull request by insang1**
I have  written code for invalid inputs

### CODE REVIEW

File: src/main/java/sig/oss/Grader.java [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+	final static int MAX_SCORE = 50;
 	static public Grade calculate_grade(int mid, int fin) {
 		int tot;
 		Grade grade=Grade.F;
-		tot = mid+fin;
-		if (tot>=80) grade=Grade.A;
-		else if (tot>=70)grade=Grade.B;
-		else if (tot>=60)grade=Grade.C;
-		else grade=Grade.F;
+		if (check_score(mid) && check_score(fin)) {
```
Comment (ischung) [inline src/main/java/sig/oss/Grader.java]: @insang1 add test codes for invalid inputs

Comment (insang1) [inline src/main/java/sig/oss/Grader.java]: I will add more tests for invalid inputs

Comment (ischung) [inline src/main/java/sig/oss/Grader.java]: @insang1 hurry up

File: src/test/java/sig/oss/GraderTest.java [hunk 2]
```diff
@@ -10,12 +10,16 @@ public void testGradeA() {
...
 
 	}
-		@Test
+	@Test
 	public void testGradeF() {
 		Grade g = Grader.calculate_grade(0,0);
 		assertEquals(Grade.F, g);
 
 	}
+	@Test(expected=InvalidInputException.class)
```
Comment (insang1) [inline src/test/java/sig/oss/GraderTest.java]: @ischung Review new added JUnit test for invalid input

Comment (ischung) [inline src/test/java/sig/oss/GraderTest.java]: @insang1 :+1:

**Pull request merged**